Platforms/QemuSbsaPkg/QemuSbsaPkg.dsc

```diff
@@ -843,6 +842,11 @@
   gEfiSecurityPkgTokenSpaceGuid.PcdTpm2HashMask|0
 !endif
 
+  #

```

Comment:
It's already zero, but if it's not in the DSC don't we hit autogen issues?

Reply:
It should work with the default from the dec.

What issue did you run into? Something like PCD referenced in a package not declared in an INF or something like that?

Reply:
For example the following issue:

`
INFO - In file included from /home/chfernal/mu_tiano_platforms/Build/QemuSbsaPkg/DEBUG_GCC5/AARCH64/QemuSbsaPkg/Library/SbsaQemuLib/SbsaQemuLib/DEBUG/AutoGen.h:17,
INFO -                  from <command-line>:
INFO - /home/chfernal/mu_tiano_platforms/Platforms/QemuSbsaPkg/Library/SbsaQemuLib/SbsaQemuMem.c: In function ‘SbsaQemuLibConstructor’:
INFO - /home/chfernal/mu_tiano_platforms/MU_BASECORE/MdePkg/Include/Library/PcdLib.h:511:38: error: implicit declaration of function ‘_PCD_SET_MODE_64_S_PcdMmBufferBase’ [-Werror=implicit-function-declaration]
INFO -   511 | #define PcdSet64S(TokenName, Value)  _PCD_SET_MODE_64_S_##TokenName    ((Value))
INFO -       |                                      ^~~~~~~~~~~~~~~~~~~
INFO - /home/chfernal/mu_tiano_platforms/Platforms/QemuSbsaPkg/Library/SbsaQemuLib/SbsaQemuMem.c:89:15: note: in expansion of macro ‘PcdSet64S’
INFO -    89 |   PcdStatus = PcdSet64S (PcdMmBufferBase, CurBase + NewSize - PcdGet64 (PcdMmBufferSize));
INFO -       |               ^~~~~~~~~
`

This is in the changed file which does declare ArmPkg/ArmPkg.dec in it's INF

Reply:
Yeah, I think those are because this PCD supports both fixed and dynamic type. So a default value statement is needed to indicate the build script that this is used as a dynamic PCD.

Reply:
That is because the .dec provides different PCD option types - FixedAtBuild, Dynamic, and PatchableInModule.

In this case, the DSC needs to select what type the platform is actually going to use. So, keeping it in the DSC makes sense to make that selection.